blockchain/types/transaction.go

```diff
@@ -829,6 +829,26 @@ func TxDifference(a, b Transactions) (keep Transactions) {
 	return keep
 }
 
+// FilterTransactionWithBaseFee returns a list of transactions for each account that filters transactions
+// that are greater than or equal to baseFee.
+func FilterTransactionWithBaseFee(pending map[common.Address]Transactions, baseFee *big.Int) (map[common.Address]Transactions, error) {
```

Comment:
Where this method is used?

Reply:
@jeongkyun-oh It is used in `work/worker.go` where filtering txs lower than `basefee`